src/xrCore/xrDebug.cpp

```diff
@@ -446,30 +446,34 @@ void xrDebug::GatherInfo(char* assertionInfo, size_t bufferSize, const ErrorLoca
     int nptrs = backtrace(array, 20);     // get void*'s for all entries on the stack
     char **strings = backtrace_symbols(array, nptrs);
 
-    if(strings)
+    if (strings)
     {
+        size_t demangledBufSize = 0;
```

Comment:
Should be initialized with braces: `size_t demangledBufSize{};`

Reply:
Are you sure this is more readable?

Reply:
Less readable, but it is my code style.

Well, let it as is for now then, don't want to force it right now =)